landingai/common.py

```diff
@@ -212,3 +217,23 @@ def decode_bitmap_rle(
         map_number = encoding_map[map_letter]
         flat_mask.extend([int(map_number)] * int(num))
     return flat_mask
+
+
+def get_prediction_bounding_box(prediction: Prediction) -> Optional[BoundingBox]:
+    """
+    Returns the prediction bounding box coordinates for compatible Predictions.
+    If prediction is not compatible with BoundingBox extraction, returns None.
```

Comment:
it'd be helpful to add a note in the doc to warn user about the caveat of converting OCR to BoundingBox, i.e. the potential precision loss.
i'd even add a warning log to make it more obvious.

Reply:
Added note about precision loss.